src/SelectTrait.php

```diff
@@ -39,4 +39,38 @@ public function selectShouldNotHaveOption($select, $option) {
     }
   }
 
+  /**
+   * @Then select :select should have option :option selected
+   */
+  public function selectShouldHaveOptionSelected($select, $option) {
+    $selectElement = $this->getSession()->getPage()->findField($select);
```

Comment:
please refactor to use `selectShouldHaveOption()` (add returning of the element to `selectShouldHaveOption()` ). and then use `getAttribute()`.